firefox_desktop/explores/newtab_interactions.explore.lkml

```diff
@@ -17,4 +17,182 @@ explore: newtab_interactions {
+        newtab_interactions.channel,
+        newtab_interactions.country_code,
+        newtab_interactions.activity_segment,
+        newtab_interactions.pocket_enabled,
+        newtab_interactions.pocket_is_signed_in,
+        newtab_interactions.pocket_sponsored_stories_enabled,
+        newtab_interactions.topsites_enabled,
+        countries.tier,
+      ]
+      measures: [
+        newtab_interactions.visits,
+        newtab_interactions.clients,
+      ]
+      filters: [newtab_interactions.submission_date: "after 2022-07-01"]
+    }
+
+    materialization: {
+      sql_trigger_value:
+      SELECT
+        MAX(last_modified_time)
+      FROM
+        moz-fx-data-shared-prod.telemetry_derived.INFORMATION_SCHEMA.PARTITIONS
+      WHERE
+        table_name = "newtab_interactions_v1";;
+      increment_key: newtab_interactions.submission_date
+      increment_offset: 1
+    }
+  }
+
+  aggregate_table: rollup__newtab_interactions_pocket_per_day {
+    query: {
+      dimensions: [
+        newtab_interactions.submission_date,
+        newtab_interactions.channel,
+        newtab_interactions.country_code,
+        newtab_interactions.pocket_story_position,
+        countries.tier,
+      ]
+      measures: [
+        newtab_interactions.visits_with_pocket_impressions,
+        newtab_interactions.visits_with_pocket_saves,
+        newtab_interactions.visits_with_pocket_clicks,
+        newtab_interactions.visits_with_organic_pocket_impressions,
+        newtab_interactions.visits_with_organic_pocket_saves,
+        newtab_interactions.visits_with_organic_pocket_clicks,
+        newtab_interactions.visits_with_sponsored_pocket_impressions,
+        newtab_interactions.visits_with_sponsored_pocket_saves,
+        newtab_interactions.visits_with_sponsored_pocket_clicks,
+
+        newtab_interactions.sum_pocket_impressions,
+        newtab_interactions.sum_pocket_saves,
+        newtab_interactions.sum_pocket_clicks,
+        newtab_interactions.sum_organic_pocket_impressions,
+        newtab_interactions.sum_organic_pocket_saves,
+        newtab_interactions.sum_organic_pocket_clicks,
+        newtab_interactions.sum_sponsored_pocket_impressions,
+        newtab_interactions.sum_sponsored_pocket_saves,
+        newtab_interactions.sum_sponsored_pocket_clicks,
+
+        newtab_interactions.clients_with_pocket_impressions,
+        newtab_interactions.clients_with_pocket_saves,
+        newtab_interactions.clients_with_pocket_clicks,
+        newtab_interactions.clients_with_organic_pocket_impressions,
+        newtab_interactions.clients_with_organic_pocket_saves,
+        newtab_interactions.clients_with_organic_pocket_clicks,
+        newtab_interactions.clients_with_sponsored_pocket_impressions,
+        newtab_interactions.clients_with_sponsored_pocket_saves,
+        newtab_interactions.clients_with_sponsored_pocket_clicks,
+      ]
+      filters: [newtab_interactions.submission_date: "after 2022-07-01"]
```

Comment:
Can we also also add a filter here to only include countries with Pocket?

Reply:
+1 if its possible